apps/frontend/src/libs/helpers/date-helpers.ts

```diff
@@ -0,0 +1,11 @@
+const DATE_PART_INDEX = 0;
+
+const formatDateForInput = (dateString: null | string): string => {
+	if (!dateString) {
+		return "";
+	}
+
+	return new Date(dateString).toISOString().split("T")[DATE_PART_INDEX] ?? "";
```

Comment:
to make it look better I would suggest to change the destructuring approach to
`
const [date] = new Date(dateString).toISOString().split("T")
return date ?? ""
`